pkg/cli/cli.go

```diff
@@ -87,6 +87,7 @@ const (
 	buildIDFlag               = "build_id"
 	buildTargetFlag           = "build_target"
 	localImageFlag            = "local_image"
+	localArtifactsFlag        = "local_artifacts"
```

Comment:
nit: Add _file to the flag to make it clear that it points to a file describing the artifacts, not the artifacts themselves.

Reply:
no longer needed after changing to flags.